privacyidea/lib/tokens/totptoken.py

```diff
@@ -678,39 +678,33 @@ def get_default_settings(cls, params, logged_in_user=None,
             return ret
         (role, username, userrealm, adminuser, adminrealm) = determine_logged_in_userparams(logged_in_user,
                                                                                             params)
-        hashlib_pol = policy_object.get_action_values(
-            action="totp_hashlib",
-            scope=role,
-            user=username,
-            realm=userrealm,
-            adminuser=adminuser,
-            adminrealm=adminrealm,
-            client=client_ip,
-            unique=True)
+        hashlib_pol = Match.generic(g, scope=role,
+                                    action="totp_hashlib",
```

Comment:
please check if we can use constants here

Reply:
see above